int32 PAUSE=0
int32 ROBOT_POSE=1
int32 TRACK=2
int32 SEGMENT=3
int32 REWIND=4

int32 mode
---
bool success